src/images/svg/symbols/bitcoin_flag.svg

```diff
@@ -0,0 +1,20 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
+    <g fill="none" fill-rule="evenodd">
```

Comment:
Very small nitpick here, but could you also minify this file? Rest of the PR looks good to me 👍